static/app/views/issueList/utils.spec.tsx

```diff
@@ -4,33 +4,31 @@ import {getTabs} from './utils';
 
 describe('getTabs', () => {
   it('should enable/disable tabs for escalating-issues', () => {
```

Comment:
Should the test name here and below get updated as well?

Reply:
Yeah actually I'll rewrite those since I think they should have been deleted when the features GA'd